clustered/integration-test/src/test/java/org/ehcache/clustered/management/ClusteringManagementServiceTest.java

```diff
@@ -263,15 +263,17 @@ public void test_D_server_capabilities_exposed() throws Exception {
     // tms entity
 
     managerCapabilities = readTopology().activeServerEntityStream().filter(serverEntity -> serverEntity.is(tmsServerEntityIdentifier)).findFirst().get().getManagementRegistry().get().getCapabilities().toArray(new Capability[0]);
-    assertThat(managerCapabilities.length).isEqualTo(3);
+    assertThat(managerCapabilities.length).isEqualTo(5);
 
-    assertThat(managerCapabilities[0].getName()).isEqualTo("OffHeapResourceSettings");
-    assertThat(managerCapabilities[1].getName()).isEqualTo("OffHeapResourceStatistics");
-    assertThat(managerCapabilities[2].getName()).isEqualTo("StatisticCollectorCapability");
+    assertThat(managerCapabilities[0].getName()).isEqualTo("DataRootSettings");
```

Comment:
What do the `DataRoot` capabilities do in an OSS system given that they are EE features?

Reply:
no - we had a meeting with @jhouserizer and you were there and in this meeting we all decided that it was ok to move the data-root resource to oss when moving dynamic config plus all the utility classes (we talked about them in this meeting too).